--- /usr/libexec/SmartCardServices/drivers/ifd-ccid.bundle/Contents/Info.plist	2014-03-11 10:12:40.000000000 +0100
+++ /usr/libexec/SmartCardServices/drivers/ifd-ccid.bundle/Contents/Info.plist_updated	2014-03-11 10:30:34.000000000 +0100
@@ -104,6 +104,8 @@
 
 	<key>ifdVendorID</key>
 	<array>
+	        <string>0x1050</string>
+		<string>0x1050</string>
 		<string>0x08E6</string>
 		<string>0x08E6</string>
 		<string>0x08E6</string>
@@ -237,6 +239,8 @@
 
 	<key>ifdProductID</key>
 	<array>
+	        <string>0x0111</string>
+		<string>0x0112</string>
 		<string>0x2202</string>
 		<string>0x3437</string>
 		<string>0x3438</string>
@@ -370,6 +374,8 @@
 
 	<key>ifdFriendlyName</key>
 	<array>
+	        <string>Yubikey NEO OTP+CCID</string>
+		<string>Yubikey NEO CCID</string>
 		<string>Gemplus Gem e-Seal Pro</string>
 		<string>Gemplus GemPC Twin</string>
 		<string>Gemplus GemPC Key</string>
