--- /usr/libexec/SmartCardServices/drivers/ifd-ccid.bundle/Contents/Info.plist_not_edited	2014-03-13 12:45:29.000000000 +0100
+++ /usr/libexec/SmartCardServices/drivers/ifd-ccid.bundle/Contents/Info.plist_updated	2014-09-03 11:37:13.000000000 +0200
@@ -104,6 +104,10 @@
 
 	<key>ifdVendorID</key>
 	<array>
+	        <string>0x1050</string>
+		<string>0x1050</string>
+		<string>0x1050</string>
+		<string>0x1050</string>
 		<string>0x08E6</string>
 		<string>0x08E6</string>
 		<string>0x08E6</string>
@@ -237,6 +241,10 @@
 
 	<key>ifdProductID</key>
 	<array>
+	        <string>0x0111</string>
+		<string>0x0112</string>
+		<string>0x0115</string>
+		<string>0x0116</string>
 		<string>0x2202</string>
 		<string>0x3437</string>
 		<string>0x3438</string>
@@ -370,6 +378,10 @@
 
 	<key>ifdFriendlyName</key>
 	<array>
+	        <string>Yubikey NEO OTP+CCID</string>
+		<string>Yubikey NEO CCID</string>
+		<string>Yubikey NEO U2F+CCID</string>
+		<string>Yubikey NEO OTP+U2F+CCID</string>
 		<string>Gemplus Gem e-Seal Pro</string>
 		<string>Gemplus GemPC Twin</string>
 		<string>Gemplus GemPC Key</string>
