From 6dd501199c1323a49f2e64e75c886a1d0b652577 Mon Sep 17 00:00:00 2001
From: Dag Heyman <dag@yubico.com>
Date: Thu, 20 Feb 2020 18:05:59 +0100
Subject: [PATCH] setup.py: add typical snapcraft location of PCSC

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 939e60f..ba54c7c 100755
--- a/setup.py
+++ b/setup.py
@@ -76,7 +76,7 @@
     platform_swig_opts = ['-DPCSCLITE']
     platform_sources = []
     platform_libraries = []
-    platform_include_dirs = ['/usr/include/PCSC', '/usr/local/include/PCSC']
+    platform_include_dirs = ['/root/stage/usr/include/PCSC']
     platform_extra_compile_args = []    # ['-ggdb', '-O0']
     platform_extra_link_args = []   # ['-ggdb']
 
